ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift

```diff
@@ -0,0 +1,381 @@
+  var title: String
+  
+  var body: some View {
+    Button {
+      action()
+    } label: {
+      Text(title)
+        .font(.subheadline)
+        .foregroundStyle(Color(braveSystemName: .textPrimary))
+        .frame(maxWidth: .infinity, alignment: .leading)
+      
+      Image(braveSystemName: "leo.arrow.small-down")
+        .foregroundStyle(Color(braveSystemName: .iconDefault))
+        .padding(.leading)
+    }
+    .padding(12.0)
+    .background {
+      RoundedRectangle(cornerRadius: 8.0, style: .continuous)
+        .stroke(Color(braveSystemName: .iconDefault), lineWidth: 0.5)
+    }
+    .background(Color(braveSystemName: .containerBackground))
+    .clipShape(RoundedRectangle(cornerRadius: 8.0, style: .continuous))
+  }
+}
+
+private struct AIChatDropdownMenu<Item>: View where Item: RawRepresentable, Item.RawValue: StringProtocol, Item: Identifiable {
+  @Binding
+  var selectedIndex: Int
+  var items: [Item]
+  
+  var body: some View {
+    VStack(spacing: 0.0) {
+      ForEach(Array(items.enumerated()), id: \.offset) { index, option in
+        Button {
+          selectedIndex = index
+        } label: {
+          Text(option.rawValue)
+            .font(.subheadline)
+            .foregroundStyle(Color(braveSystemName: .textPrimary))
+            .frame(maxWidth: .infinity, alignment: .leading)
+          
+          if index == selectedIndex {
+            Image(braveSystemName: "leo.check.normal")
+              .foregroundStyle(Color(braveSystemName: .iconDefault))
+          }
+        }
+        .padding()
+        
+        if index != items.count {
+          Color(braveSystemName: .dividerSubtle)
+            .frame(height: 1.0)
+        }
+      }
+    }
+    .background {
+      RoundedRectangle(cornerRadius: 8.0, style: .continuous)
+        .stroke(Color(braveSystemName: .iconDefault), lineWidth: 0.5)
+    }
+    .background(Color(braveSystemName: .containerBackground))
+    .clipShape(RoundedRectangle(cornerRadius: 8.0, style: .continuous))
+    .shadow(color: .black.opacity(0.25), radius: 10.0, x: 0.0, y: 2.0)
+  }
+}
+
+private struct AIChatDropdownView: View {
+  enum Options: String, CaseIterable, Identifiable {
+    case notHelpful
+    case notWorking
+    case other
+    
+    var id: Self { self }
+
+    var rawValue: String {
+      switch self {
+      case .notHelpful:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      case .notWorking:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      case .other:
+        return Strings.AIChat.feedbackOptionTitleNotHelpful
+      }
+    }
+  }
+  
+  @Binding
+  var selectedIndex: Int
+  
+  @State
+  private var showMenu = false
+
+  var body: some View {
+    VStack {
+      HStack(spacing: 0.0) {
+        Text(Strings.AIChat.feedbackOptionsViewTitle)
+          .font(.caption)
+          .fontWeight(.semibold)
+          .foregroundStyle(Color(braveSystemName: .textPrimary))
+        
+        Text("*")
```

Comment:
I assume we dont want to use an actual asterisk here?  Better to use an SF Symbol with `.imageScale(.small)`, otherwise make sure you mark it as verbatim (`Text(verbatim: "*")`)